src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -36,8 +36,8 @@ public AddCommand(String name,
                       String email, boolean isEmailPrivate,
                       String address, boolean isAddressPrivate,
                       Set<String> tags) throws IllegalValueException {
-        final Set<Tag> tagSet = new HashSet<>();
-        for (String tagName : tags) {
+    	final Set<Tag> tagSet = new HashSet<>();
```

Comment:
Very good, you have located the right place to handle tag addition.

However, you should make use of the new Tagging class, e.g. a set of <Tagging> instead of <Tag>. <Tagging> associate a Tag with a Person.